packages/worker/scripts/build-worker-string.js

```diff
@@ -5,7 +5,14 @@ const fs = require('fs')
 
 const bundlePath = path.resolve(__dirname, '../bundle')
 const workerBundlePath = path.join(bundlePath, 'worker.js')
-const workerStringPath = path.join(bundlePath, 'workerString.ts')
 const workerBundleContent = fs.readFileSync(workerBundlePath, { encoding: 'utf-8' })
 
-fs.writeFileSync(workerStringPath, `export const workerString = ${JSON.stringify(workerBundleContent)}`)
+writeOutput('cjs', `exports.workerString = ${JSON.stringify(workerBundleContent)}\n`)
+writeOutput('esm', `export const workerString = ${JSON.stringify(workerBundleContent)}\n`)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
const workerString = `${JSON.stringify(workerBundleContent)}\n`
writeOutput('cjs', `exports.workerString = ${workerString}`)
writeOutput('esm', `export const workerString = ${workerString}`)
```